### DISCUSSION TITLE
How to get the YOLO badge?

### DISCUSSION THREAD

**Discussion by A-PRIVATE1** [community/community#140575] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do I get the new YOLO GitHub achievement?

**Comment by Ali-Barzegari-d** [comment, marked as answer]
You just have to merge a pull request without code review.

https://github.com/Schweinepriester/github-profile-achievements

**Reply by A-PRIVATE1** [reply]
thank you ali
